index.d.ts

```diff
@@ -147,4 +147,5 @@ declare class Minipass<
   [Symbol.asyncIterator](): AsyncGenerator<RType, void, void>
 }
 
-export = Minipass
+export default Minipass;
+export = Minipass;
```

Comment:
As I understand it, there should be the following solution:

```ts
export default Minipass;
export = Minipass;
``` 

Because right now the current solution from this PR willwill generate the following error:

```ts
An export assignment cannot be used in a module with other exported elements.ts(2309)
```

Related issues:

* https://github.com/DefinitelyTyped/DefinitelyTyped/pull/64329

Reply:
Yes, exactly.
Also had this issue when trying to Upgrade `glob` to the most recent version.

With `glob` using the non-fixed version of this package, TypeScript will report exactly this error.